src/core/core.scale.js

```diff
@@ -1282,15 +1287,26 @@ class Scale extends Element {
 			ctx.textBaseline = 'middle';
 			ctx.textAlign = item.textAlign;
 
+			if (useTextWithStroke) {
```

Comment:
I still think `tickFont` should contain the `lineWidth` and `strokeStyle` inside this loop. Out of the loop you don't actually know if its a minor or major tick that we are drawing.

Reply:
Take a look:

![image](https://user-images.githubusercontent.com/676681/72175126-a9551000-33db-11ea-933a-d6a4d615191e.png)

Inside just this loop: https://github.com/chartjs/Chart.js/pull/6787/files#diff-f0d8f86fecab28f396793911b0e9f36dL1272

```
		for (i = 0, ilen = items.length; i < ilen; ++i) {
			const item = items[i];
			const tickFont = item.font;

console.log('tickFont: ' + tickFont.lineWidth + ' - optionTicks: ' + optionTicks.lineWidth);
```